fastplotlib/graphics/selectors/_linear.py

```diff
@@ -227,11 +229,19 @@ def __init__(
             axis=axis,
         )
 
+    @property
+    def selection(self) -> LinearSelectionFeature:
+        """
+        The current selection in world space. These bounds will NOT necessarily correspond to the
+        indices of the data that are under the selection.
+        """
+        return self._selection
+
     def _setup_ipywidget_slider(self, widget):
         # setup ipywidget slider with callbacks to this LinearSelector
-        widget.value = int(self.selection())
+        widget.value = int(self._selection())
         widget.observe(self._ipywidget_callback, "value")
-        self.selection.add_event_handler(self._update_ipywidget)
+        self._selection.add_event_handler(self._update_ipywidget)
```

Comment:
once `_selection` is defined in the construction just access it using the property.